ci/Jenkinsfile.android

```diff
@@ -59,7 +59,8 @@ pipeline {
           print "Running ${btype} build!"
           cmn.ci.abortPreviousRunningBuilds()
           /* Cleanup and Prep */
-          mobile.prep(btype)
+          cmn.prepNixEnvironment()
+          cmn.prep(btype)
```

Comment:
We might as well put `cmn.prepNixEnvironment()` inside of `cmn.prep()` if we call it every time.

Reply:
Good point